atm_challenge.md

```diff
@@ -0,0 +1,49 @@
+# ATM challenge
+
+## Introduction
+
+This challenge is to write a software for an ATM, where a few rules, must be applied in order to
+withdraw money.
```

Comment:
This challenge is to write software for an ATM where a few rules must be applied to
withdraw money.